server/planning/events/events_template.py

```diff
@@ -0,0 +1,250 @@
+        'DELETE': 'planning_event_management',
+        'PATCH': 'planning_event_management',
+        'PUT': 'planning_event_management'
+    }
+    _event_fields = {
+        'slugline': {
+            'type': 'string',
+            'required': False,
+            'readonly': True
+        },
+        'name': {
+            'type': 'string',
+            'required': False,
+            'readonly': True
+        },
+        'definition_short': {
+            'type': 'string',
+            'required': False,
+            'readonly': True
+        },
+        'definition_long': {
+            'type': 'string',
+            'required': False,
+            'readonly': True
+        },
+        'internal_note': {
+            'type': 'string',
+            'required': False,
+            'readonly': True
+        },
+        'ednote': {
+            'type': 'string',
+            'required': False,
+            'readonly': True
+        },
+        'links': {
+            'type': 'list',
+            'readonly': True
+        },
+        'occur_status': {
+            'type': 'dict',
+            'schema': {
+                'qcode': {
+                    'type': 'string'
+                },
+                'name': {
+                    'type': 'string'
+                },
+                'label': {
+                    'type': 'string'
+                },
+            },
+            'readonly': True
+        },
+        'files': {
+            'type': 'list',
+            'schema': Resource.rel('events_files'),
+            'readonly': True
+        },
+        'calendars': {
+            'type': 'list',
+            'schema': {
+                'type': 'dict',
+                'schema': {
+                    'qcode': {
+                        'type': 'string'
+                    },
+                    'name': {
+                        'type': 'string'
+                    },
+                    'is_active': {
+                        'type': 'boolean'
+                    }
+                }
+            },
+            'readonly': True
+        },
+        'location': {
+            'type': 'list',
+            'schema': {
+                'type': 'dict'
+            },
+            'readonly': True
+        },
+        'event_contact_info': {
+            'type': 'list',
+            'schema': Resource.rel('contacts'),
+            'readonly': True
+        },
+        'subject': {
+            'type': 'list',
+            'schema': {
+                'type': 'dict'
+            },
+            'readonly': True
+        }
+    }
+    schema = {
+        'template_name': {
+            'type': 'string',
+            'required': True,
+            'empty': False,
+            'unique': True
+        },
+        'based_on_event': Resource.rel(
+            'events',
+            type=metadata_schema[config.ID_FIELD]['type'],
+            embeddable=False,
+            required=True
+        ),
+        **_event_fields
```

Comment:
I would do it more like: `'data': {'type': 'dict', 'schema': _event_fields}`

Reply:
Ahh, got it. Sure, it makes sense.